### ISSUE TITLE
Support for PasswdSafe Sync

### ISSUE THREAD

**Issue by kyelbek** [tejado/Authorizer#11]
I'm editing HID patch for my device right now and to that day I was using [PasswdSafe by Jeff Harris](https://play.google.com/store/apps/details?id=com.jefftharris.passwdsafe).
Since your project is based on the same sources, can you maybe make it compatible with great addon, for version I metioned earlier, called [PasswdSafe Sync (albo by the same person)](https://play.google.com/store/apps/details?id=com.jefftharris.passwdsafe.sync)?
For now, when you open Authorizer you will got security errors when have this apk addon installed immediately after openning your app. It is [open-sourced](https://sourceforge.net/p/passwdsafe/code/ci/default/tree/sync/) too but if you can't make it compatible (by compiling it probably for your fork) then just please remove part of code that is calling it when starting Authorizer because it is impossible to use in my case. If it is only matter of few simple tweaks then I will try to build it myself.
[Screenshot of error](https://i.imgur.com/dZOmFSu.png)

**Comment by tejado** [comment]
Were you successful with the HID patch?

So I tried to remove most of the Sync stuff to reduce the code. Also my intitial idea is still an offline device so I didn't need it. Feel free to create a PR to merge the Sync part into Authorizer again.

Which Authorizer version caused that error on your screenshot?
As the current versions available in F-Droid and Play store are not really up to date, I guess the latest code would work for you. I will check it and update the stores.

**Comment by kyelbek** [comment]
I patched HID without a problem and it's working with permissive SELinux but I don't know how to configure SELinux policies to make it work in Enforcing state so I opened a ticket [here](https://github.com/pelya/android-keyboard-gadget/issues/146).
I personally like the idea of syncing databases because I often experimenting with my device and sometimes I'm forgetting to copy some data to PC when changing ROMs. I just want to configure and forget. I'm noob with programming bigger projects and using GIT but I will try to do something myself so there can be two variants of Authorizer. At least I will learn something new.
I'm probably using version from F-Droid (0.2.4beta). I downloaded apk from GitHub to my PC at first but my MTP broke that time while trying cd emulation patch so I just downloaded it from F-D. I didn't know that there is newer version of Authorizer. I remember checking your app 6+ months ago while I was thinking about unlocking my other phone just to use this brilliant idea and I've encountered this bug too.
As I see version 0.2.5 is still having same issue so I will build newer code and check if it's still colliding with sync and report back to you. I also encountered another bug but it might be already fixed.
Thanks for your reply.
